### ISSUE TITLE
The github calendar widget is not working well

### ISSUE THREAD

**Issue by herculeskan** [joelshepherd/tabliss#641]
![Github Calendar screenshoot](https://github.com/joelshepherd/tabliss/assets/80991265/02bb70db-0e09-4090-bf4c-5368398a9ec1)

It's display dates however that's the only thing that has been display in a long time... It is a way to fix this issue? I attempted with different browsers  and still the same

**Comment by thedaveCA** [comment]
https://github.com/joelshepherd/tabliss/issues/620#issuecomment-1637130183 has a fix. Or did, it seems to have disappeared completely as of today.

**Comment by herculeskan** [comment]
Thank you, I just paste the code, however, yeah, it's seems like disappeared today, I'm going to wait until troubleshooting c:

**Comment by herculeskan** [comment]
yep, changing the css worked, however I think It could be better if would working well as default

**Comment by thedaveCA** [comment]
Last night it wasn't working at all, we're back to it working with the CSS, which is something. But I agree, the issue should be addressed via the other bug rather than requiring manual CSS.

**Comment by minding-myown-business** [comment]
Just got here to open this exact issue - it's very buggy for me as well. It doesn't even show my real contribution history - it says i have not contributed at all this year, when I have actually (at least a bit)
![image](https://github.com/joelshepherd/tabliss/assets/150592702/93d37571-127f-4add-a3a4-c0536501026a)
![image](https://github.com/joelshepherd/tabliss/assets/150592702/3078759a-df81-4fbb-8af3-c8acac5a3820)

**Comment by the-wright-jamie** [comment]
My fork, [Tab Nine](https://github.com/the-wright-jamie/tab-nine), has fixed this (mostly, still a little weird at unusual vertical screen resolutions) 

Related issues/commits
- https://github.com/the-wright-jamie/tab-nine/issues/28
- cf1c0b9fbc6dfbe42fdea09e5084173544225612
